ecl/eclcmd/eclcmd_core.cpp

```diff
@@ -416,7 +422,7 @@ class EclCmdPublish : public EclCmdWithEclTarget
         StringBuffer wuid;
         if (optObj.type==eclObjWuid)
             wuid.set(optObj.value.get());
-        else if (!doDeploy(*this, client, optMsToWait, optTargetCluster.get(), optName.get(), &wuid, NULL, optNoArchive))
+        else if (!doDeploy(*this, client, optMsToWait, optTargetCluster.get(), optName.get(), &wuid, NULL, optNoArchive, true, true, false))
```

Comment:
Why not allow a user to set the protect here?

Reply:
I debated these two changes because I wanted to try and make 8.0.0 but you are right these two commands can have protect added as well.  I will add them.